templates/vue/src/components/modals/NodeModal/forms/ContentForm/VideoForm.vue

```diff
@@ -40,11 +41,12 @@
       <b-col cols="6" md="8">
         <b-form-input
           v-show="useKaltura"
-          v-model="node.typeData.kalturaId"
+          :value="node.typeData.kalturaId"
           data-qa="node-video-kaltura-id"
           name="text-input"
           placeholder="Enter Kaltura video ID"
           required
+          @input="update('typeData.kalturaId', $event)"
```

Comment:
```suggestion
          @update="update('typeData.kalturaId', $event)"
```

Bootstrap has very weird v-models, this is one of them.

Reply:
This is just for your information. I've made some changes to this file (and reverting back to using v-model) since we don't need the full node, only mediaURL and kalturaId.

Reply:
OK, thanks for letting me know!